pages/en/lb2/Model-definition-JSON-file.md

```diff
@@ -101,6 +101,13 @@ Properties are required unless otherwise designated.
       </td>
     </tr>
     <tr>
+      <td>forceId</td>
+      <td>Boolean</td>
+      <td>
+        The default value is set to true whenever a model uses an auto-generated ID. If it is set to true it prevents users from setting the auto-id value manually.
```

Comment:
Yes, we should copy `Model-definition-JSON-file.md` to the `lb3` folder.  Please do that and make any changes required.